core/trino-spi/src/main/java/io/trino/spi/metrics/Distribution.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.spi.metrics;
+
+public interface Distribution
+{
+    double getTotal();
```

Comment:
Why `double`?

Reply:
Because `TDigest#getCount` returns `double` - so I used the same type here.

Reply:
That's because TDigest and QDigest are designed to support exponential decay, which results in fractional weights over time. I don't think we need that here.